src/components/ProfileCard/ProfileCard.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { Card, Avatar, Typography, Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { greyScale } from '../../style-library/types/theme.helpers';
+import { SIZE_CONFIGS, VARIANT_CONFIGS } from './ProfileCard.constants';
+
+import type {
+  StyledProfileCardProps,
+  ProfileCardSize,
+  ProfileCardVariant,
+} from './ProfileCard.types';
+
+export const StyledCard = styled(Card, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $size }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+
+  return {
+    padding: theme.spacing(sizeConfig.padding / 8),
+    borderRadius: theme.spacing(2),
+    boxShadow: theme.shadows[2],
+    transition: theme.transitions.create(['box-shadow', 'transform'], {
+      duration: theme.transitions.duration.short,
+    }),
+    '&:hover': {
+      boxShadow: theme.shadows[4],
+      transform: 'translateY(-2px)',
+    },
+    [theme.breakpoints.down('sm')]: {
+      padding: theme.spacing(2),
+    },
+    [theme.breakpoints.up('md')]: {
+      padding: theme.spacing(sizeConfig.padding / 8),
+    },
+  };
+});
+
+export const StyledProfileContainer = styled(Box, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $variant }) => {
+  const variantConfig = VARIANT_CONFIGS[$variant];
+
+  return {
+    display: 'flex',
+    flexDirection: variantConfig.direction,
+    alignItems: 'center',
+    gap: theme.spacing(2),
+    width: '100%',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+      gap: theme.spacing(1.5),
+    },
+    [theme.breakpoints.up('md')]: {
+      flexDirection: variantConfig.direction,
+    },
+  };
+});
+
+export const StyledAvatar = styled(Avatar, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<{ $size: ProfileCardSize }>(({ theme, $size }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+
+  return {
+    width: sizeConfig.avatarSize,
```

Comment:
Avatar width/height set to `sizeConfig.avatarSize` will be treated as raw numbers (px). This violates the project's 'no hardcoded px' guideline and couples the styles to specific pixel values.

**Sugerencia:**
```typescript
Make `avatarSize` a spacing-unit value in constants and use `width: theme.spacing(sizeConfig.avatarSize)` and `height: theme.spacing(sizeConfig.avatarSize)`. Example change:

// constants: avatarSpacing: 6
width: theme.spacing(sizeConfig.avatarSpacing),
height: theme.spacing(sizeConfig.avatarSpacing),
```